.github/project.yml

```
@@ -1,3 +1,3 @@
 release:
-  current-version: 1.1.0
-  next-version: 1.2.0-SNAPSHOT
+  current-version: 2.0.0.Alpha4
```

```suggestion
  current-version: 2.0.0.Alpha
```

This is our first Alpha :D

---

Don't you think it's better to follow Quarkus' Alpha numbers?

---

I don't think so, this is our release train, not theirs.